exchanges/coinbasepro/coinbasepro_wrapper.go

```diff
@@ -284,9 +312,36 @@ func (c *CoinbasePro) FetchAccountInfo(ctx context.Context, assetType asset.Item
 	return acc, nil
 }
 
-// UpdateTickers updates the ticker for all currency pairs of a given asset type
-func (c *CoinbasePro) UpdateTickers(_ context.Context, _ asset.Item) error {
-	return common.ErrFunctionNotSupported
+// UpdateTickers updates all currency pairs of a given asset type
+func (c *CoinbasePro) UpdateTickers(ctx context.Context, assetType asset.Item) error {
```

Comment:
This function is usually used for assets that allow batching requests, so we might be able to revert this, as it wont be used anyway.